docs/security/access-control.md

```diff
@@ -152,16 +152,16 @@ You can also create a dedicated role for your business by API or the client.
 
 ### Permission privileges
 
-| Name          | Supports Securable Object | Operation              |
-|---------------|---------------------------|------------------------|
-| MANAGE_GRANTS | Metalake                  | grant or revoke a role |
+| Name          | Supports Securable Object | Operation                                                                         |
+|---------------|---------------------------|-----------------------------------------------------------------------------------|
+| MANAGE_GRANTS | Metalake                  | Grant or revoke roles for a user or group. Grant or revoke privileges for a role. |
```

Comment:
Manages roles granted to or revoked from the user or group, and privileges granted to or revoked from the role.

Reply:
Changed.